src/ui/components/SideBar/index.tsx

```diff
@@ -181,6 +189,14 @@ export const SideBar: FC = () => {
           onClick={handleAddServerButtonClicked}
         >+</AddServerButtonLabel>
       </AddServerButton>
+      <BackButton>
+        <BackButtonLabel
+          tooltip={t('sidebar.backButton')}
+          onClick={handleBackButtonClicked}
```

Comment:
`onClickEvent` should be used in button component, not in label of the button

- FULL DISCLOSURE

  - i'm not very familiar with this project architeture, but for what i understand so far we are using [Fuselage](https://github.com/RocketChat/Rocket.Chat.Fuselage) design system that's already had a button component, if we need to create a custom button shouldn't this be an abstraction of the [design system `Button` component](https://github.com/RocketChat/Rocket.Chat.Fuselage/blob/a7fe68a411ccd6c80aa2c1f95b4fabcb495f0aee/packages/fuselage/src/components/Button/Button.stories.tsx)?